.github/CODEOWNERS

```diff
@@ -1,13 +1,6 @@
-
-# Python
```

Comment:
This change is showing here even though it was also merged into /feature/skosgi/termGradeTrigger - I need to investigate how to get this diff to display only the real changes...

Reply:
@kdy618 I think I figured out how to resolve this, but it will result in a new branch and pull request. Not a big deal - feel free to work on the other changes I've identified here in the current branch. We should schedule a video call tomorrow and I can show you how I think we can fix this - I'll add something to the calendar now.

Reply:
@robertbwatson , that would be great!